src/components/search-body/search-sidebar/SearchSidebar.vue

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+import CategorySidebar from './category-sidebar/CategorySidebar.vue'
+import { defineComponent } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useParamsStore } from '@/stores/params-store'
+
+export default defineComponent({
+    name: 'SearchSidebar',
+    components: {
+        CategorySidebar
+    },
+    setup(props, { emit }) {
+        const { t } = useI18n()
+        const paramsStore = useParamsStore()    
+
+        const handleOnClickResetFilters = (): void => {
+            emit('toggle-sidebar')
```

Comment:
Vì sao khi reset filter thì e lại toggle sidebar. E mô tả thêm luồng hoạt động của toggle sidebar này nha

Reply:
em dùng toggle-sidebar trong trường hợp responsive cho màn hình điện thoại. 
Khi người dùng nhấn vào 'reset filter', toàn bộ trạng thái sẽ được xóa và kết quả sẽ được trả về ngay để hiển thị cho người dùng nên đoạn này em mới dùng toggle-sidebar để close cái sidebar lại

[screen-capture (4).webm](https://github.com/user-attachments/assets/d05ae98c-f9bb-417b-88ba-38eb9a624704)